skate/source/includes/_leaderboard.md

```diff
@@ -212,7 +212,7 @@ Parameter | Required | Type | Default | Description
 --------- | -------- | ---- | ------- | -----------
 `user_id`   | yes    | Integer |      | The ID of the user to create a leaderboard entry for.
 `score`     | yes    | Integer |      | The score value for the entry in the leaderboard.
+`tag`       | no     | String |       | An identification tag for a leaderboard entry.
 `radius`    | yes    | Integer |      | The number of scores to return above and below the user's score.
```

Comment:
I thought that too initially, but then I wanted to group the parameters relating to the "adding" part separate from the "listing" part. That felt like a more important distinction to draw in the docs.